### DISCUSSION TITLE
The window post succeeds, but the page still fails to display

### DISCUSSION THREAD

**Discussion by lbj2004032** [filecoin-project/lotus#9725] [Storage Provider] [answered]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [X] lotus daemon - chain sync
- [X] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [X] Other

### Lotus Version

```text
lotus-miner version 1.18.0+mainnet+git.bd10bdf99
```

### Describe the Bug

![1669292501936](https://user-images.githubusercontent.com/101935091/203783466-c237a938-f437-42a8-a091-4c7772923d93.png)
![1669292501936(1)](https://user-images.githubusercontent.com/101935091/203783473-c45c0dcb-efcd-49dd-9a89-2eea2a2e591a.png)

### Logging Information

```text
no error log
```

### Repo Steps

1. Run '...'
2. Do '...'
3. See error '...'
...

**Comment by Reiers** [comment]
Hi @lbj2004032 

Can you run:
`lotus-miner proving check --slow --only-bad 14`

Thanks!

**Reply by lbj2004032** [reply]
The v1.16.0 windowpost I used failed yesterday. I did not find the reason. I directly upgraded to v.1.18.0


$ lotus-miner proving check --slow --only-bad 14
deadline  partition  sector  status

**Comment by Reiers** [comment, marked as answer]
Moved to Discussions 👍  

The reason why I'm asking is that windowPost for deadline 14 was **_not_** successful.
```
Method
SubmitWindowedPoSt
Value
0 FIL
Status
ErrIllegalArgument
```
https://filfox.info/en/message/bafy2bzacec42httow4xpsqbmpeq75okwdnqqq2qs6npqtraesbwj24jqeaclo 

So it's either a bad sector that causes issues, or there where a issue with funding.

**Reply by lbj2004032** [reply]
but windowpost successful

**Reply by Reiers** [reply]
It was not: `Status: ErrIllegalArgument`
That means your last post message landed onChain too late.
